miteD/middleware/api.py

```diff
@@ -9,6 +14,24 @@
 from miteD.service.client import RemoteService
 
 
+TTL_CHECK_IN_INTERVAL = os.getenv("TTL_CHECK_IN_INTERVAL") or 15
```

Comment:
not important and just my OCD but in the future one can resort to this kind of syntax `TTL_CHECK_IN_INTERVAL = os.getenv("TTL_CHECK_IN_INTERVAL", 15)`  i.e. the default can be specified in the call's second 
 argument. See https://docs.python.org/3/library/os.html#os.getenv

Reply:
For sure! I didn't realize that but makes sense.I'll change it.

Reply:
not required, just was a heads up

Reply:
Actually I'd opt for that change as when the env var is defined and has value = 0 it would still be overwritten by 15, which is not the intent here.
I'm not assuming 0 value would make or not make sense in this case.